source/adapter.js

```diff
@@ -741,7 +741,9 @@ if (navigator.mozGetUserMedia) {
   // the minimum version still supported by adapter.
   webrtcMinimumVersion = 12;
 
-  getUserMedia = navigator.getUserMedia;
```

Comment:
Why not just use 
window.getUserMedia = navigator.mozGetUserMedia.bind(navigator);
navigator.getUserMedia = window.getUserMedia;
like everywhere else ???